lib/HARToPostmanCollectionMapper.js

```diff
@@ -32,9 +32,20 @@ const DEFAULT_COLLECTION_NAME = 'Generated from HAR',
   {
     groupEntriesByOption
   } = require('./utils/groupingHelper'),
+
+  PSEUDO_HEADERS = [':method', ':scheme', ':authority', ':path'],
```

Comment:
@diyaeninan Do we have any source that these are the only pseudo-headers? According to [RFC](https://www.rfc-editor.org/rfc/rfc7540#section-8.1.2.1), pseudo headers are something that starts with `:`.

We should check for the presence of `:` at start of header name instead of only few hard-coded value in this case.

Reply:
@VShingala The pseudo-header fields for a request are mentioned below [here](https://www.rfc-editor.org/rfc/rfc7540#section-8.1.2.3)

> Pseudo-header fields are not HTTP header fields.  Endpoints MUST NOT
   generate pseudo-header fields other than those defined in this
   document.

RFC also mentions that no other headers should be generated as pseudo headers

Reply:
@diyaeninan Got it, makes sense. Thanks for the reference and also upon cheking more HAR files, found that these 4 are the only ones that are included.